README.md

```diff
@@ -2,11 +2,20 @@
 
 ⚠️ **Warning: this project is still very alpha**
 
-This project is:
+React component variations are:
 * A format called "variations" for specifying React component **_examples_** and providing **_documentation_** for them.
 * Tools for finding, validating, and iterating over variations.
+* A collection of "consumers" for doing useful things with variations.
 
-A variation looks like:
+## Table of contents
+
+- [Variations](#variations)
+- [Consumers](#consumers)
+- [Contribution Guidelines](#contribution-guidelines)
+
+## Variations
+
+Variations are defined in files we call variation providers. These are standard Javascript modules that return an object with specific fields. For example:
```

Comment:
Would it make sense to elaborate on these "specific fields" that a variation provider's return object needs to have?

Reply:
Yeah

Reply:
- JavaScript
 - they're not actually standard JS modules, they're node modules, which atm are just commonJS
```suggestion
Variations are defined in files we call variation providers. A "variation provider" is a node module that exports a function, that returns a "descriptor object" with specific fields. For example:
```